### PR TITLE
Fixed invalid PMT and PAT version increments

### PR DESCRIPTION
**Pull request by asticode**

### CODE REVIEW

File: muxer_test.go [hunk 1]
```diff
@@ -52,11 +53,18 @@ func TestMuxer_generatePAT(t *testing.T) {
 	assert.Equal(t, MpegTsPacketSize, muxer.patBytes.Len())
 	assert.Equal(t, patExpectedBytes(0, 0), muxer.patBytes.Bytes())
 
-	// to check version number increment
+	// Version number shouldn't change
```
Comment (barbashov) [inline muxer_test.go:56]: It would be great to do the same test for PMT. Otherwise the code looks good :)

Comment (asticode) [reply muxer_test.go:56]: you're right I've added the same test for PMT 👍

Comment (barbashov) [reply muxer_test.go:56]: I think now it's ready to merge :)

Comment (barbashov) [reply muxer_test.go:56]: Oh, you've done it already :)

**Pull request merged**